docs/catena portal guide/CatenaPortalGuide.md

```diff
@@ -0,0 +1,95 @@
+
+
+Table of Contents
+  - [Table of Contents](#table-of-contents)
+  - [Starting Point](#starting-point)
+  - [Find Your Company](#find-your-company)
+  - [Create Technical User](#create-technical-user)
+  - [Register Connector](#register-connector)
+    
+  - [NOTICE](#notice)
+
+## Starting Point
+
+Before anything else, your company needs to registered on the Catena Portal, for that you can create an issues here: https://github.com/eclipse-tractusx/portal-frontend-registration
+
```

Comment:
This is quite misleading, as Tractus-X just provides the reference implementation for Catena-X and opening an issue to one a repo in TX will not lead to getting registered for Catena-X. I suggest you link to our documentation https://github.com/eclipse-tractusx/portal-assets/blob/v1.8.0/docs/user/01.%20Onboarding/index.md

Reply:
Fixed via commit:
https://github.com/catenax-ng/tx-demand-capacity-mgmt/commit/ac8a3231eeb2ff6ad0f8f699b861bc547cf6e921

Reply:
@ClosedSourcerer the comment appears not be fixed with catenax-ng@ac8a323

Reply:
When looking at the rich diff you can see, that I inserted the portal user guide prominently.
I however did not remove the the previous content completely, but rather reduced its prominense by moving it into an expandable "details" function.

My reasoning is that I am planning on expanding/correcting the documentation to have a main 100% generic path and than an optional paths for specific scenarios (mainly Windows in SMEs that can utilize docker desktop with incuring license costs)

![image](https://github.com/eclipse-tractusx/demand-capacity-mgmt/assets/35833350/c8bd8b0f-11e5-40be-b678-864bc0f778dc)

If this is not a valid solution for you, please tell me and I will remove rows 14 to 93 alltogether.

Reply:
@ClosedSourcerer this comment is about the following wrong statement:
![image](https://github.com/eclipse-tractusx/demand-capacity-mgmt/assets/46191613/d89d8485-6ecc-4641-a1d4-937b2c70e9c5)
This statement is wrong because Tractus-X just provides the reference implementation for Catena-X and opening an issue to https://github.com/eclipse-tractusx/portal-frontend-registration will not lead to getting registered for Catena-X. Therefore this statement needs to be removed.

Reply:
Thank you very much.

I removed the statement and replaced it with a general statement, as well as a table of the currently known OpCos and Portals.

https://github.com/catenax-ng/tx-demand-capacity-mgmt/commit/e850ba92f0779d344013293dbf961d036a96c68e